2024-05-17 22:48:00,453 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Configure stats pid to 3525
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from /content/drive/MyDrive/MiniGPT/wandb/settings
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-17 22:48:00,454 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-17 22:48:00,455 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-17 22:48:00,455 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-05-17 22:48:00,455 INFO    MainThread:3525 [wandb_init.py:_log_setup():520] Logging user logs to /content/drive/MyDrive/MiniGPT/wandb/run-20240517_224800-k13aaqxt/logs/debug.log
2024-05-17 22:48:00,455 INFO    MainThread:3525 [wandb_init.py:_log_setup():521] Logging internal logs to /content/drive/MyDrive/MiniGPT/wandb/run-20240517_224800-k13aaqxt/logs/debug-internal.log
2024-05-17 22:48:00,456 INFO    MainThread:3525 [wandb_init.py:init():560] calling init triggers
2024-05-17 22:48:00,456 INFO    MainThread:3525 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-17 22:48:00,456 INFO    MainThread:3525 [wandb_init.py:init():585] re-initializing run, found existing run on stack: ldbcsyaq
2024-05-17 22:48:00,459 INFO    MainThread:3525 [wandb_run.py:_finish():2103] finishing run zxzucla/dl2_proj3/ldbcsyaq
2024-05-17 22:48:00,459 INFO    MainThread:3525 [jupyter.py:save_history():445] not saving jupyter history
2024-05-17 22:48:00,459 INFO    MainThread:3525 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-17 22:48:00,460 INFO    MainThread:3525 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-17 22:48:00,460 INFO    MainThread:3525 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-17 22:48:00,461 INFO    MainThread:3525 [wandb_run.py:_restore():2326] restore
2024-05-17 22:48:00,461 INFO    MainThread:3525 [wandb_run.py:_restore():2332] restore done
2024-05-17 22:48:04,082 INFO    MainThread:3525 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-17 22:48:04,083 INFO    MainThread:3525 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-17 22:48:04,097 INFO    MainThread:3525 [wandb_run.py:_footer_sync_info():3953] logging synced files
2024-05-17 22:48:04,105 INFO    MainThread:3525 [wandb_init.py:init():610] starting backend
2024-05-17 22:48:04,106 INFO    MainThread:3525 [wandb_init.py:init():614] setting up manager
2024-05-17 22:48:04,111 INFO    MainThread:3525 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-17 22:48:04,111 INFO    MainThread:3525 [wandb_init.py:init():622] backend started and connected
2024-05-17 22:48:04,131 INFO    MainThread:3525 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-17 22:48:04,436 INFO    MainThread:3525 [wandb_init.py:init():711] updated telemetry
2024-05-17 22:48:04,476 INFO    MainThread:3525 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-17 22:48:05,321 INFO    MainThread:3525 [wandb_run.py:_on_init():2396] communicating current version
2024-05-17 22:48:05,466 INFO    MainThread:3525 [wandb_run.py:_on_init():2405] got version response 
2024-05-17 22:48:05,466 INFO    MainThread:3525 [wandb_init.py:init():795] starting run threads in backend
2024-05-17 22:48:05,956 INFO    MainThread:3525 [wandb_run.py:_console_start():2374] atexit reg
2024-05-17 22:48:05,956 INFO    MainThread:3525 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2024-05-17 22:48:05,957 INFO    MainThread:3525 [wandb_run.py:_redirect():2294] Wrapping output streams.
2024-05-17 22:48:05,957 INFO    MainThread:3525 [wandb_run.py:_redirect():2319] Redirects installed.
2024-05-17 22:48:05,958 INFO    MainThread:3525 [wandb_init.py:init():838] run started, returning control to user process
2024-05-17 22:50:12,886 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-17 22:50:12,890 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Configure stats pid to 3525
2024-05-17 22:50:12,890 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from /content/drive/MyDrive/MiniGPT/wandb/settings
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-17 22:50:12,891 INFO    MainThread:3525 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-05-17 22:50:12,892 INFO    MainThread:3525 [wandb_init.py:_log_setup():520] Logging user logs to /content/drive/MyDrive/MiniGPT/wandb/run-20240517_225012-wybt3r0d/logs/debug.log
2024-05-17 22:50:12,892 INFO    MainThread:3525 [wandb_init.py:_log_setup():521] Logging internal logs to /content/drive/MyDrive/MiniGPT/wandb/run-20240517_225012-wybt3r0d/logs/debug-internal.log
2024-05-17 22:50:12,892 INFO    MainThread:3525 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fda53c83dc0>
2024-05-17 22:50:12,893 INFO    MainThread:3525 [wandb_init.py:init():560] calling init triggers
2024-05-17 22:50:12,893 INFO    MainThread:3525 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-17 22:50:12,893 INFO    MainThread:3525 [wandb_init.py:init():585] re-initializing run, found existing run on stack: k13aaqxt
2024-05-17 22:50:12,895 INFO    MainThread:3525 [wandb_run.py:_finish():2103] finishing run zxzucla/dl2_proj3/k13aaqxt
2024-05-17 22:50:12,896 INFO    MainThread:3525 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-17 22:50:12,896 INFO    MainThread:3525 [wandb_run.py:_restore():2326] restore
2024-05-17 22:50:12,897 INFO    MainThread:3525 [wandb_run.py:_restore():2332] restore done
